packages/gluestick/src/renderer/helpers/getAssetsLoader.js

```diff
@@ -27,8 +27,21 @@ module.exports = function getAssetsLoader(
       });
     };
 
+    function isPolyfillNeeded() {
+      return !window['fetch'];
+    }
+
     document.addEventListener('DOMContentLoaded', function() {
-      loadVendorThenEntry();
+      if (isPolyfillNeeded()) {
+        loadjs('https://cdn.polyfill.io/v2/polyfill.min.js', {
```

Comment:
Is there any way we could host the polyfill through GlueStick… I'm wondering what happens if polyfill.io goes down

Reply:
As we discussed, if we served this file directly from GlueStick, this might work: https://cdnjs.cloudflare.com/ajax/libs/babel-polyfill/6.26.0/polyfill.min.js